src/EventListener/StoreFbcSubscriber.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusFacebookPlugin\EventListener;
+
+use Symfony\Component\HttpKernel\KernelEvents;
+
+final class StoreFbcSubscriber extends AbstractSubscriber
```

Comment:
The facebook click id should be saved in a cookie and have a lifetime of the default lifetime that Facebook uses which I think is 28 days. This value should be configurable